near-sdk/src/collections/legacy_tree_map.rs

```diff
@@ -18,6 +20,7 @@ use crate::IntoStorageKey;
 /// - `above`/`below`:          O(log(N))
 /// - `range` of K elements:    O(Klog(N))
 ///
+#[deprecated(since = "4.1.0", note = "Use near_sdk::collections::TreeMap")]
```

Comment:
noting that the since "4.1.0" is a bit unideal, but given we don't really have a plan what will be the next minor/major version, and no one uses this, it's probably fine

Reply:
So yeah, how do we feel about depreciations? Are the major release only, or do we just do it whenever?